lib/use_cases/save_rent_detail.rb

```diff
@@ -0,0 +1,120 @@
+class RentDetail
+  def initialize(rent_detail_gateway:, fixed_data_gateway:)
+    @rent_detail_gateway = rent_detail_gateway
+    @fixed_data_gateway = fixed_data_gateway.get_rc_uplift
```

Comment:
> so fixed_data_gateway is just a placeholder until a method exists in fixed_data_gateway so that here we can have fixed_data_gateway.method_to_get_rc_uplift?

@rebeccafitzsimmons1 I called the `fixed_data_gateway.get_rc_uplift` method in the initialize method.

Reply:
@jpmaHub ah yes I see. Maybe the vairable name `@fixed_data_gateway` is a bit misleading?